main.go

```diff
@@ -544,6 +544,48 @@ func (session *Session) DeleteTag(tag Tag) ([]byte, error) {
 	return session.delete(TogglAPI, path)
 }
 
+// GetClients returns a list of clients for the current account
+func (session *Session) GetClients() ([]Client, error) {
```

Comment:
How would you feel about `(clients []Client, err error)` as the return type, and then using the `clients` variable in the `return` statements in the function?

Reply:
actually makes the code more readable - changed accordingly